README.md

```diff
@@ -1870,7 +1870,7 @@ authClient.token.getWithPopup()
 
```

Comment:
I recommend creating a PENDING section in the changelog for any non-draft PR, so we never lose track of what has and hasn't been released should the PR get merged.